### PR TITLE
DROOLS-3895: [DMN Designer] TextAreas can no longer be multi-line

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/DROOLS-3895

Part of an ensemble:

- https://github.com/kiegroup/appformer/pull/684
- https://github.com/kiegroup/kie-wb-common/pull/2615

### CODE REVIEW

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by manstis** [comment]
@jomarko What's the status of your [PR](https://github.com/kiegroup/kie-wb-common/pull/2590)?

**Comment by jomarko** [comment]
@manstis Hi, I implemented review commentaries there, need re-review from all incorporated, will mark people in ensemble.

File: uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/keyboard/KeyDownHandlerCommon.java [hunk 1]
```diff
@@ -35,37 +34,83 @@
...
+
+        reset();
+    }
+
+    private void reset() {
+        gridCell.destroyResources();
+        gridPanel.setFocus(true);
+        gridLayer.batch();
+    }
+/**/
```
Comment (jomarko) [inline uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/keyboard/KeyDownHandlerCommon.java]: I think we do not need this comment, am I missing something?

Comment (manstis) [reply uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/keyboard/KeyDownHandlerCommon.java]: @jomarko Nope, you're not missing anything IDEA's `Reformat code...` context menu is next to `Comment with Block Comment` entry and I simply selected the wrong one! Anyway, comment removed in new commit :-)

**Comment by manstis** [comment]
@karreiro If you get a chance to look at this before you go away.... :-)

**Pull request merged**